llama-index-integrations/llms/llama-index-llms-bedrock/llama_index/llms/bedrock/utils.py

```diff
@@ -111,14 +113,27 @@ def completion_to_anthopic_prompt(completion: str) -> str:
 
 
 class AnthropicProvider(Provider):
-    max_tokens_key = "max_tokens_to_sample"
+    max_tokens_key = "max_tokens"
 
     def __init__(self) -> None:
         self.messages_to_prompt = messages_to_anthropic_prompt
         self.completion_to_prompt = completion_to_anthopic_prompt
 
+    def get_text_from_stream_response(self, response: dict) -> str:
+        if response["type"] == "content_block_delta":
+            return response["delta"]["text"]
+        else:
+            return ""
+
     def get_text_from_response(self, response: dict) -> str:
-        return response["completion"]
+        return response["content"][0]["text"]
+
+    def get_request_body(self, prompt: Sequence[Dict], inference_parameters: dict):
+        return {
+            "messages": prompt,
+            "anthropic_version": "bedrock-2023-05-31",  # Required by AWS.
```

Comment:
This will overload the value if explicitly passed.

Maybe convert to:

```py
"anthropic_version": inference_parameters.get("anthropic_version", "bedrock-2023-05-31"),  # Required by AWS.
```

Reply:
Done.